pshtt/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM lambci/lambda:build-python3.6
+FROM lambci/lambda:build-python3.7
```

Comment:
Please make an issue _somewhere_ to migrate these Lambda configurations. The Docker images from lambci haven't been updated in almost three years.

Reply:
Please see cisagov/lambda_functions#49.